workflow-templates/dreact-ci.yml

```diff
@@ -0,0 +1,62 @@
+name: DReact CI
+on:
+  pull-request:
+    types: [closed]
+
+jobs:
+  main:
+    name: main
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged == true
+    env: # Update for each app
+      APP_SLUG: app-slug
+      APP_TITLE: App Title
+    steps:
+      - name: Checkout Source
+        uses: actions/checkout@v2
+        with:
+          path: main
+      - name: Checkout Module Repo
+        uses: actions/checkout@v2
+        with:
+          repository: "BrownUniversity/dreact"
+          path: dreact
+          token: ${{ secrets.API_TOKEN_GITHUB }} # PAT
+      - uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+      - name: Install App Dependencies
+        run: npm ci
+        working-directory: main
+      - name: Build App
+        run: npm run build
+        working-directory: main
+      - name: Copy App to Module
+        run: cp main/dist/${{ env.APP_SLUG }}.js dreact/apps/${{ env.APP_SLUG }}.js
+      - name: Create Pull Request in Module Repo
+        id: pull-request
+        uses: brownuniversity/create-pull-request@v1.0.15
```

Comment:
Should this be a major version? Or update the v1 tag to point to this release?

Reply:
I think the idea is that the v1 tag should be rolling and always point to the latest v1.x release. If we need to use a more specific version that is always possible via the more specific tags.